### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'width' - error fter push CREATE MASK

### ISSUE THREAD

**Issue by numbereleven-a** [limithit/Mask2Background#8]
version:  •  python: 3.10.10  •  torch: 2.0.0+cu118  •  xformers: 0.0.20  •  gradio: 3.29.0  •  checkpoint: [c54f90e540]

File "c:\Temp_\_programs\Stable Def\venv\lib\site-packages\gradio\routes.py", line 408, in run_predict
    output = await app.get_blocks().process_api(
  File "c:\Temp_\_programs\Stable Def\venv\lib\site-packages\gradio\blocks.py", line 1315, in process_api
    result = await self.call_function(
  File "c:\Temp_\_programs\Stable Def\venv\lib\site-packages\gradio\blocks.py", line 1043, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "c:\Temp_\_programs\Stable Def\venv\lib\site-packages\anyio\to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "c:\Temp_\_programs\Stable Def\venv\lib\site-packages\anyio\_backends\_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "c:\Temp_\_programs\Stable Def\venv\lib\site-packages\anyio\_backends\_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "C:\Temp_\_programs\Stable Def\extensions\Mask2Background\scripts\inpaint_anything.py", line 43, in wrapper
    res = func(*args, **kwargs)
  File "C:\Temp_\_programs\Stable Def\extensions\Mask2Background\scripts\inpaint_anything.py", line 117, in select_mask
    ret_image = create_mask(input_image)
  File "C:\Temp_\_programs\Stable Def\extensions\Mask2Background\scripts\inpaint_anything.py", line 43, in wrapper
    res = func(*args, **kwargs)
  File "C:\Temp_\_programs\Stable Def\extensions\Mask2Background\scripts\inpaint_anything.py", line 86, in create_mask
    for h in range(img.width):
AttributeError: 'NoneType' object has no attribute 'width'

**Comment by limithit** [comment]
![image](https://github.com/limithit/Mask2Background/assets/7675726/323d60a1-b709-4d46-9a6f-13696e6e8106)
What is your version of AUTOMATIC 1111?

**Comment by limithit** [comment]
Your dependencies were not installed successfully perhaps

**Comment by eastone11** [comment]
version: [## 1.4.1] i have black screens
![bf201ba50e3d145b532e648121920e3e](https://github.com/limithit/Mask2Background/assets/139941473/b5eaad35-6312-42f7-9932-3d177cb180cd)

**Comment by eastone11** [comment]
AUTOMATIC 1111?
yes

**Comment by limithit** [comment]
Obviously, you didn't read the readme.md, which warns that images need to be in png format and contain transparent pixels in order to work, and your image has a background, so you get an error!

**Comment by limithit** [comment]
![image](https://github.com/limithit/Mask2Background/assets/7675726/7b7782a9-c072-43b5-9879-aa7157fc8642)
Masek's is called a clear zone.

**Comment by eastone11** [comment]
Obviously, but not obvious.
![7c02275f2688c3da4e2e3d27fb436236](https://github.com/limithit/Mask2Background/assets/139941473/562a8178-4e98-497a-b6b5-a68d1dcb4865)
